pyflakes/checker.py

```diff
@@ -1234,6 +1234,9 @@ def handleNodeLoad(self, node):
         if name == '__module__' and isinstance(self.scope, ClassScope):
             return
 
+        if not PY2 and name == '__qualname__' and isinstance(self.scope, ClassScope):
```

Comment:
~technically this should be (3, 3) but we don't support 3.x (x<4) so this should be fine